src/www/status_dhcp_leases.php

```diff
@@ -48,9 +48,11 @@ function adjust_gmt($dt)
     }
 
     foreach ($dhcpd as $dhcpditem) {
-        $dhcpleaseinlocaltime = $dhcpditem['dhcpleaseinlocaltime'];
-        if ($dhcpleaseinlocaltime == "yes") {
-            break;
+        if (isset($dhcpditem['dhcpleaseinlocaltime'])) {
```

Comment:
you can inline the trailing code for $dhcpleaseinlocaltime:

if (isset($dhcpditem['dhcpleaseinlocaltime']) && $dhcpditem['dhcpleaseinlocaltime'] == "yes") {
         $ts = strtotime($dt . " GMT");
         return strftime("%Y/%m/%d %I:%M:%S%p", $ts);
}